ext_emconf.php

```diff
@@ -40,15 +40,15 @@
     'clearCacheOnLoad' => false,
     'lockType' => '',
     'version' => '5.0.0',
```

Comment:
I suggest version 5.1.0 for this maintenance release to support TYPO3 8.7 AND 9.5.

Reply:
done